ge_releaser/git.py

```diff
@@ -36,8 +35,22 @@ def checkout_and_pull_trunk(self) -> None:
         self._git.git.checkout(self._trunk)
         self._git.git.pull(self._remote, self._trunk)
 
+    def check_for_untracked_files(self) -> bool:
+        return bool(self._git.git.untracked_files)
+
+    def verify_no_untracked_files(self) -> None:
+        if self.check_for_untracked_files():
+            raise ValueError("There are untracked files. Please make sure to run this step with a clean repo.")
+
     def stage_all_and_commit(self, message: str) -> None:
-        self._git.git.add(".")
+        files_to_add = (
+            "docs/docusaurus/docs/changelog.md",
+            "docs/docusaurus/docs/components/_data.jsx",
+            "docs/docusaurus/docusaurus.config.js",
+            "docs_rtd/changelog.rst",
+            "great_expectations/deployment_version",
```

Comment:
We have a number of constants for this - could we use them? Lemme find the file

Reply:
https://github.com/greatexpectationslabs/ge_releaser/blob/main/ge_releaser/constants.py